roadmap.md

```diff
@@ -28,12 +28,11 @@ the library into 3 pieces.
    on Android, iOS and any programming language.
 
 # Phase 2 - In progress 🚧
+- 🚧 Migration from `Hyperledger libindy` to its successor [vdr-tools](https://gitlab.com/evernym/verity/vdr-tools).
```

Comment:
Although most will probably understand what you mean, `Hyperledger libindy` is not stricly correct nomenclature and slightly confusing - I think `libindy` is accurate enough.  Whether and in what sense it is a successor is debatable (it is not a successor for some other projects), I would rather avoid the term "successor".